schema/3.0.0/schema.md

```diff
@@ -503,7 +503,7 @@ Curators MUST annotate the following columns in the `obs` dataframe:
     </tr>
     <tr>
       <th>Value</th>
-        <td>categorical with <code>str</code> categories. If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be either a HANCESTRO term or <code>"unknown"</code> if unavailable. <br><br>Otherwise, for all other organisms this MUST be <code>"na"</code>.
+        <td>categorical with <code>str</code> categories. If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be either a HANCESTRO term, <code>"unknown"</code> if unavailable, or <code>"multiethnic"</code> if more than one ethnicities are reported. <br><br>Otherwise, for all other organisms this MUST be <code>"na"</code>.
```

Comment:
Slight editorial changes:

this MUST be either a HANCESTRO term, <code>"multiethnic"</code> if multiple ethnicities are reported, or <code>"unknown"</code> if unavailable.

or

this MUST be either a HANCESTRO term, <code>"multiethnic"</code> if more than one ethnicity is reported, or <code>"unknown"</code> if unavailable.

Reply:
addressed in 51649bd